.github/workflows/build.yml

```diff
@@ -79,7 +79,7 @@ jobs:
       - name: Install dependencies
         run: |
           brew update
-          brew install autoconf automake pkg-config libtool python freetype fribidi little-cms2 luajit libass ffmpeg meson
+          brew install autoconf automake pkg-config libtool python freetype fribidi little-cms2 lua@5.1 libass ffmpeg meson
```

Comment:
try `luajit-openresty`

Reply:
This PR actually originally tried luajit-openresty but somehow `dependency('luajit` ..` in meson didn't pick it up so the build failed. I didn't bother to see what it was actually called, but if it's not a standard name it kind of rules it out imo. Or maybe I just did something wrong I dunno.

Reply:
Oh actually looking at the logs, for this to work the path would need to be exported to `PKG_CONFIG_PATH`. That's kind of meh.